Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /opt/nlisim
+COPY nlisim /opt/nlisim/nlisim
+COPY setup.py setup.cfg /opt/nlisim/
+COPY .git /opt/nlisim/.git
+RUN apt update \
+    && apt install -y git libgl1 \
```

Comment:
As a separate matter, it'd be nice to see if we can remove the OpenGL dependency, if we're only installing `vtk` for its I/O capabilities.

Reply:
FWIW, you need libgl just to import vtk.

```
$ python -c 'import vtk'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/bin/env/lib/python3.6/site-packages/vtk.py", line 30, in <module>
    all_m = importlib.import_module('vtkmodules.all')
  File "/bin/env/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/bin/env/lib/python3.6/site-packages/vtkmodules/all.py", line 29, in <module>
    from .vtkRenderingOpenGL2 import *
ImportError: libGL.so.1: cannot open shared object file: No such file or directory

```